Modules/_abc.c

```diff
@@ -258,6 +252,7 @@ _abc__get_dump(PyObject *module, PyObject *self)
 static int
 compute_abstract_methods(PyObject *self)
 {
+    _Py_IDENTIFIER(__abstractmethods__);
```

Comment:
Why don't we define these identifiers at the top level?

Reply:
I prefer local variable when it's used only in one function.
If you prefer top level, I'll revert unnecessary changes.

Reply:
I usually move them to the top level as it's hard to track if a variable like this is used in one or in many functions. Really up to you though, I'm OK about this change either way.